examples/src/Viewer.tsx

```diff
@@ -369,17 +366,17 @@ class Viewer extends React.Component<InputParams, ViewerState> {
             });
     }
 
-    public clearPendingFile() {
-        this.setState({ filePending: null });
-    }
-
```

Comment:
quick sanity check: this looks like something important, where the code wanted to know that there was a file in-flight?  is it really ok to remove it and within scope of this PR?  or is this dead code that was never used?